homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                  (id       int     UNIQUE      PRIMARY KEY     NOT NULL,
</pre><p>Primary is already unique</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='117'/>